### ISSUE TITLE
query-based variables does not work with Grafana 11.3.0

### ISSUE THREAD

**Issue by arvidsnet** [akamai/akamai-insights-datasource#11]
after Grafana update to 11.3.0 query-based variables stopped to work. Screenshots with variable config and error attached. Please let me know if some more details are needed.

![dashboard](https://github.com/user-attachments/assets/a32c26e1-6251-4d82-b869-b0ef564b5fa2)
![query](https://github.com/user-attachments/assets/1332548a-0fc4-456f-a8fd-3aa3f040c9ec)

**Comment by ppozniak-akamai** [comment]
Do you have any errors in the javascript console related to the issue?

**Comment by arvidsnet** [comment]
yes, there is following error in console:
```
15:48:20.480 SceneVariableSet updateAndValidate error Error: Couldn't create a query runner for datasource akamai-insights-datasource
    yv createQueryVariableRunner.js:113
    getValueOptions QueryVariable.js:73
    RxJS 5
    Angular 18
    RxJS 16
    _updateNextBatch SceneVariableSet.js:127
    _onActivate SceneVariableSet.js:35
    _internalActivate SceneObjectBase.js:161
    _internalActivate SceneObjectBase.js:160
    activate SceneObjectBase.js:193
    _internalActivate SceneObjectBase.js:170
    activate SceneObjectBase.js:193
    Jt SceneComponentWrapper.js:40
    React 3
    Q scheduler.production.min.js:13
    Dt scheduler.production.min.js:14
    7463 scheduler.production.min.js:14
    Webpack 17
SceneVariableSet.js:161:12
```

**Comment by mikeZuber** [comment]
@arvidsnet for now you can switch from Scenes to old dashboards view 
![Screenshot 2024-11-06 at 4 27 53 PM](https://github.com/user-attachments/assets/9b4da0ad-31d0-4fdf-a09d-0357954fc9fe)

Fix for scenes will be released in next version.

**Comment by mikeZuber** [comment]
@arvidsnet Fix for Scenes API is available in version 1.1.0
